docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -1,16 +1,55 @@
+/* eslint-disable max-len */
+
+import _ from 'lodash'
 import React, { Component } from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+import { iconsByCategory as categories } from 'docs/app/utils/icons'
+import { Message } from 'stardust'
+
+const template = (content, name) => (
+`import React, { Component } from \'react\'
+import { Icon } from \'stardust\'
+
+export default class IconSet${name}Example extends Component
+  render() {
+    return(
+      <div>
+${content}
+      </div>
+    )
+  }
+}`
+)
+
+const categorySrc = ({ icons, name }) => {
+  const iconSrc = _.map(icons, icon => (`          <Icon name='${icon}' />`)).join('\n')
+  const categoryName = name.replace(/\s|&/g, '')
+  return template(iconSrc, categoryName)
+}
 
 export default class IconSetExamples extends Component {
   render() {
     return (
       <ExampleSection title='Icon Set'>
-        <ComponentExample
-          title='Web Content'
-          description='Button groups can contain conditionals'
-          examplePath='elements/Icon/IconSet/IconWebContentExample'
-        />
+        An icon set contains an arbitrary number of glyphs
+        <Message className='warning'>
+          Icons serve a very similar function to text in a page. In Semantic icons receive a special tag &lt;i&gt; which allow for an abbreviated markup when sitting along-side text.
```

Comment:
I don't think this message is relevant to our project since our Icon APIs does not use the `<i />` tag.  Unless you disagree, let's remove it.